sktime/forecasting/base/_base.py

```diff
@@ -2302,6 +2408,27 @@ def _predict_moving_cutoff(
                 )
         return _format_moving_cutoff_predictions(y_preds, cutoffs)
 
+    def _get_varnames(self, default=None, legacy_interface=True):
+        """Return variable column for DataFrame-like returns.
+
+        Developer note: currently a helper for predict_interval, predict_quantiles,
+        valid only in the univariate case. Can be extended later.
+        """
+        if legacy_interface:
+            var_name = default
+        else:
+            y = self._y
+            if isinstance(y, pd.Series):
+                var_name = self._y.name
+            elif isinstance(y, pd.DataFrame):
+                return y.columns
+            else:
+                var_name = 0
+            if var_name is None:
```

Comment:
This if-clause is unnecessary because it it should be either a pd.Series, pd.Dataframe or 0. Or do I miss something?

Reply:
if can happen that it's a `y: pd.Series` with `y.name` being `None`, that case may lead to pain.

Reply:
Ok thank you :) I approve this PR then.